src/components/SuggestedJourneys/SuggestedJourneys.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { stationsById } from "stations";
+import { JourneyParams } from "types";
+
+import styles from "./SuggestedJourneys.module.scss";
+
+type Props = {
+    journeys: JourneyParams[];
+};
+
+export function getJourneyUrl(journey: JourneyParams) {
+    return `/?from=${journey["fromStationId"]}&to=${journey["toStationId"]}&day=${journey["day"]}&time=${journey["time"]}`;
+}
+
+function getClass(time: JourneyParams) {
+    const times = time["time"];
+    if (times === undefined) {
+        return styles.midday;
+    } else if (times >= 18000 && times <= 39600) {
+        return styles.morning;
+    } else if (times > 39600 && times <= 61200) {
+        return styles.midday;
+    } else return styles.evening;
+}
+
+const SuggestedJourneys = (props: Props) => {
+    const { journeys } = props;
+    const divArr = [] as any;
+    for (let i = 0; i < journeys.length; ++i) {
+        divArr.push(
+            <div className={styles.journey}>
```

Comment:
Something I forgot to mention on the last PR: when rendering arrays of elements, React wants you to put a unique `key` prop on each element of the array [(more on that here)](https://reactjs.org/docs/lists-and-keys.html). In this case you could just add `key={i}` to this div.